task4/script.js

```diff
@@ -0,0 +1,29 @@
+const ul = document.getElementById('ul');
+
+const array = [{
```

Comment:
[ IMO ]
I think the array name should be named to point to the content.

Reply:
Thank you.
Sure, I don't know what it means if the name is "Array".
I corrected it.